### DISCUSSION TITLE
Mocked response has no body with HttpResponse.* methods

### DISCUSSION THREAD

**Discussion by SPH-0gJo** [mswjs/msw#1928] [Q&A]
I was trying to test some handler using fetch() and noticed that when I set the body in the handler with HttpResponse.text() or HttpResponse.json()  didn't get any body in response  I thought fetch() was the cause and tested fetch() with setupWorker(). response.body is undefined and response.text() in fetch callback only prints "" (empty string) in jest code. 
The only thing which is missed was response body. The other response property I had configured like statusText existed well. 
I am using 
* node 18.15.0
* msw 2.0.11
I've created an [repository which reproduces this problem](https://github.com/SPH-0gJo/msw-test-app) 

You can find test code in App.test.tsx and msw handler in handler.ts.
So, my question is: Where does the body disappear to and how to preserve it?

**Comment by kettanaito** [comment]
I believe this has been addressed in the latest releases. @SPH-0gJo, could you please update `msw` and if the issue still persists, open an issue here and provide a reproduction repo? Thank you.
